imbi/endpoints/project_ssm_configuration.py

```diff
@@ -134,6 +134,58 @@ async def get(self, *args, **kwargs) -> None:
         } for name, d in params_by_path.items()]
         self.send_response(output)
 
+    async def post(self, *args, **kwargs) -> None:
+        if not self.settings['google']['enabled']:
+            raise errors.Forbidden('Google integration is disabled',
+                                   title='Google integration is disabled')
+
+        if not self.settings['google']['integration_name']:
+            raise errors.Forbidden(
+                'No Google integration specified in configuration',
+                title='No Google integration specified in configuration')
+
+        tokens = await self.current_user.fetch_integration_tokens(
+            self.settings['google']['integration_name'])
+        if len(tokens) == 0:
+            raise errors.Forbidden(
+                'No OAuth 2.0 tokens for %s',
+                self.current_user.username,
+                title=('Not authorized to access SSM in this namespace &'
+                       ' environment'))
+        google_tokens = tokens[0]
+
+        aws_session = aioboto3.Session()
+        body = self.get_request_body()
+        for param in body['values']:
+            result = await self.postgres_execute(
+                self.GET_ROLE_ARN_SQL, {
+                    'environment': param['environment'],
+                    'namespace_id': body['namespace_id']
+                }, 'get-role-arn')
+            if not result.row:
+                raise errors.Forbidden('you cant do this')
```

Comment:
probably worth expanding this a little 😉

Reply:
oops, good catch, that was a stub I forgot to go back and elaborate